pyproject.toml

```diff
@@ -60,7 +60,7 @@ torch = [
     "torch>=2.2.0;python_version>='3.12' and python_version<'3.13'",
     "torch>=2.6.0;python_version>='3.13'",
 ]
-pomegranate = ['pomegranate>=0.15,<1.0']
+pomegranate = ['pomegranate>=1.1.2,<2.0']
 xgboost = ['xgboost>=2.1.3']
 test = [
     'sdmetrics[torch]',
```

Comment:
Since the tests won't run if pomegranate and xgboost aren't installed, let's add them to the 'test' optional dependency group.  Then we can also update the github workflows to install `.[test]` instead of `.[pomegranate,xgboost,test]`

Reply:
Yes that's a good point, done in [9d9dc59](https://github.com/sdv-dev/SDMetrics/pull/736/commits/9d9dc59f27c26e7b458a2388fa53ccb94873398b)